examples/DISCO_IOT_53L0A1_Gesture_Swipe1/DISCO_IOT_53L0A1_Gesture_Swipe1.ino

```diff
@@ -0,0 +1,182 @@
+/**
+ ******************************************************************************
+ * @file    DISCO_IOT_53L0A1_Gesture_Swipe1.ino
+ * @author  WI6LABS from AST
+ * @version V1.0.0
+ * @date    07 July 2017
+ * @brief   Arduino test application for the STMicrolectronics STM32 IOT Discovery Kit
+ *          based on FlightSense and gesture library.
+ *          This application makes use of C++ classes obtained from the C
+ *          components' drivers.
+ ******************************************************************************
```

Comment:
Perhaps you can add some more explanations about the sample. What is detected, in which range...
It is working well if we are close to the board, else, the swipe is not detected (let's say when we are at more than 50cm of the board). Is it normal ?

Same behaviour for tap detection.

Reply:
Regarding the gesture library, I’m not FlightSense expert (you should ask Imaging division for more details), but it is possible that algorithm works only at certain ranges because the sensor generates a laser cone and so the behavior of the algorithm can change when the distance of the target object changes as well. Actually, the gesture library can be also used with VL6180X component (FlightSense short range), so in my opinion it is worth to put the gesture library outside the VL53L0X library for more portability.

Reply:
So we can add this information in the sample description. Just mention to be close enough to the board.

For the gesture library, an other solution can be to isolate it in a sub-directory. It is interesting that all comes together to have a good experience on samples.

Reply:
To complete the @cparata information, the target color and indoor/outdoor situation influence the measure (see part 5 of datasheet). So the distance range is ~40-120cm.
@VVESTM I added this information in the README file and mention the gesture detection inside the library properties.